src/components/homePage/Speakers.vue

```diff
@@ -1,19 +1,136 @@
 <template>
 <div id="speakers">
-	<h2>CUTC 2020</h2>
-	<!-- TAB LAYOUT -->
+	<div class="tabs">
+		<p class="tab" :class="[currentPage == 0 ? 'underlined' : '']" @click="currentPage = 0; toggleStyle()"> Speakers </p>
+		<p class="tab" :class="[currentPage == 1 ? 'underlined' : '']" @click="currentPage = 1; toggleStyle()"> Events </p>
+	</div>
+  
+	<vueper-slides v-if="currentPage == 0">
+		<template v-slot:arrow-right>
+			<img src="../../assets/Arrow.svg"/>
+		</template>
+		<vueper-slide 
+			v-for="(slide, i) in information['slides']" 
+			:key="i" >
+			<template v-slot:content>
+                  <img :src="getImage(slide.image)" id="person" :alt="slide.alt"/>
+                {{ slide.content }}
+        <div class="vueperslide__content-wrapper">
+        </div>
```

Comment:
Shouldn't the content be within the content wrapper div? If not why is the `vueperslide__content-wrapper` div there? Also the indentation here is pretty _funky_